doc/source/whatsnew/v0.18.1.txt

```diff
@@ -9,6 +9,8 @@ We recommend that all users upgrade to this version.
 
 Highlights include:
 
+- Method chaining improvements, see :ref:`here <whatsnew_0181.enhancements.method_chain>`.
```

Comment:
can you add the examples you have above in the appropriate sections in ``indexing.rst``?